students/EricAdams/session07/html_render.py

```diff
@@ -15,18 +17,21 @@ def append(self, content):
         self.content.append(content)
 
     def render(self, file_obj, indent=''):
-        # all_content = ('<' + self.tag + '>')
-        html_tag = ('<' + self.tag + '>')
+        # what recursion level are we going in
+        Element.depth += 1
+        html_tag = (self.indent * (Element.depth - 1) + '<' + self.tag + '>\n')
         self.write_to_file(file_obj, html_tag)
         text = ""
         for each in self.content:
             try:
-                # all_content += each
                 text += each
             except TypeError:
                 each.render(file_obj)
-        html_tag = '</' + self.tag + '>'
-        text += html_tag
+        # what recursion level are at coming out
+        Element.depth -= 1
+        html_tag = '\n' + (self.indent * (Element.depth)) + \
+            '</' + self.tag + '>\n'
+        text = self.indent * (Element.depth) + text + html_tag
         self.write_to_file(file_obj, text)
 
     def write_to_file(self, file_obj, stuff_to_print):
```

Comment:
no reason to have a method for this --it's one line of code, and doesn't use self...